byzcoin/proto.go

```diff
@@ -311,6 +311,8 @@ type DownloadStateResponse struct {
 	// is generated by the server, and will be set
 	// for every subsequent reply, too.
 	Nonce uint64
+	// Total key/value pairs.
+	Total int
```

Comment:
could you add `protobuf:"opt"` for backward-compatibility?

Reply:
Done.